File: 9041/lab06/whale.py
#!/usr/bin/python3
import sys

target = sys.argv[1]
pod = 0
indi = 0

for line in sys.stdin:
    line = line.rstrip()
    whale = line.split(' ', 1)[1]
    if (whale == target):
        pod += 1
        indi += int(line.split(' ', 1)[0])
print(target, "observations:" , pod, "pods,", indi, "individuals")
        

File: 9041/lab06/whale.pl
#!/usr/bin/perl -w


$whale = $ARGV[0];

$pod = 0;
$indi = 0;

while ($line = <STDIN>) {
     chomp $line;
     @content = split / /, $line, 2;
     if ( $whale eq $content[1]) {
         $pod ++;
         $indi += $content[0];
     }
}
print $whale, " observations: ", $pod, " pods, ", $indi, " individuals\n";
